src/pybpmn/dataset.py

```diff
@@ -56,11 +58,17 @@ def __init__(
             relation_fields=relation_fields,
         )
         _logger.info(
-            "Parsed hdBPMN dataset with %d images: %s",
+            "Parsed BPMN dataset with %d images: %s",
             sum(self.split_n_imgs.values()),
             self.split_n_imgs,
         )
 
+    @abc.abstractmethod
+    def get_split_to_bpmn_paths(self) -> Dict[str, List[Path]]:
```

Comment:
Funktioniert das aktuell? Normalerweise muss in Python bei Methoden ohne Code eine Zeile mit "pass" eingefügt werden. Siehe auch: https://www.w3schools.com/python/ref_keyword_pass.asp